### PR TITLE
Uml 122 service dynamo layers in application

### PR DESCRIPTION
**Pull request by taylorsj1980**

### CODE REVIEW

File: service-api/app/src/App/src/Service/Lpa/LpaService.php [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        $this->dynamoDbClient = $dynamoDbClient;
+    }
+
+    /**
+     * Get an LPA using the ID value
+     *
+     * @param int $lpaId
+     * @return array
+     */
+    public function getById(int $lpaId) : array
```
Comment (nsmithuk) [inline service-api/app/src/App/src/Service/Lpa/LpaService.php]: Please can we make `$lpaId` a string. Although it's likely it'll always be an int, it's not an assumption we need to make.

File: service-api/app/src/App/src/Service/Lpa/LpaService.php [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        throw new NotFoundException('LPA not found');
+    }
+
+    /**
+     * Get an LPA using the share code
+     *
+     * @param int $shareCode
+     * @return array
+     */
+    public function getByCode(int $shareCode) : array
```
Comment (nsmithuk) [inline service-api/app/src/App/src/Service/Lpa/LpaService.php]: Please can you make `$shareCode` a string. i.e. the code is alphanumeric.

File: service-api/app/src/App/src/Service/Lpa/LpaService.php (hunk 1 above)
Comment (taylorsj1980) [reply service-api/app/src/App/src/Service/Lpa/LpaService.php]: Changed

File: service-api/app/src/App/src/Service/Lpa/LpaService.php (hunk 2 above)
Comment (taylorsj1980) [reply service-api/app/src/App/src/Service/Lpa/LpaService.php]: Changed

**Pull request merged**